mypy/stubgenc.py

```diff
@@ -209,6 +241,9 @@ def generate_c_type_stub(module: ModuleType,
     if all_bases[-1] is object:
         # TODO: Is this always object?
         del all_bases[-1]
+    # remove pybind11_object
```

Comment:
Really? If `pybind11` is really that unique, please elaborate more in the comment.

Reply:
I provided more specific information about this in comment. But this is part that I have doubts if this is the way it should be solved. Maybe better approach would be for pybind11 to point to an type that is actually importable.